e2e-test-server/Dockerfile

```diff
@@ -0,0 +1,47 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Build relative to root of repository i.e. `docker build --file Dockerfile --tag=$tag ..`
+
+FROM node:15-alpine as node-base
```

Comment:
Isn't v15 already deprecated?

Reply:
Yup. I'll make a PR to update all of our CI with the latest versions. Thanks!